ui/lib/pki/addon/routes/configuration/create.js

```diff
@@ -28,12 +28,4 @@ export default class PkiConfigurationCreateRoute extends Route {
       { label: 'configure' },
     ];
   }
-
-  async getOrCreateUrls(backend) {
```

Comment:
I think only ever need to findRecord because as soon as the pki mount is mounted, we can `READ` the `config/urls` endpoint. The response below is from a freshly mounted pki engine called `pki-unconfig`.
```sh
~|⇒ curl \
    --header "X-Vault-Token: root" \
    http://127.0.0.1:8200/v1/pki-unconfig/config/urls | jq

# --  response --
{
  "request_id": "e440be87-6a56-9c1d-7a62-9b78876792d9",
  "lease_id": "",
  "renewable": false,
  "lease_duration": 0,
  "data": {
    "crl_distribution_points": [],
    "enable_templating": false,
    "issuing_certificates": [],
    "ocsp_servers": []
  },
  "wrap_info": null,
  "warnings": null,
  "auth": null
}